react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -38,6 +42,15 @@ class Game extends Component {
     });
   }
 
+  async componentDidMount(){
+    this.state.isLoading = true;
```

Comment:
You need to use `setState` to modify state. Read more about it in https://reactjs.org/docs/state-and-lifecycle.html